migrations/2017_09_17_000000_create_roles_table.php

```diff
@@ -19,6 +19,8 @@ public function up()
 
                 $table->unsignedSmallInteger('sort_index')->nullable();
 
+                $table->boolean('is_admin')->default(false);
```

Comment:
Where in the code is this new column used?

If adding this is required in a project level, one can publish the migrations and customize it for their needs